openpype/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,225 @@
+from maya import cmds
+
+from openpype.hosts.maya.api import lib, plugin
+from openpype.hosts.maya.api.alembic import ALEMBIC_ARGS
+from openpype.lib import (
+    BoolDef,
+    TextDef,
+    NumberDef,
+    EnumDef,
+    UISeparatorDef,
+    UILabelDef,
+)
+
+
+def _get_animation_attr_defs(cls):
+    """Get Animation generic ddefinitions."""
+    defs = lib.collect_animation_defs()
+    defs.extend(
+        [
+            BoolDef("farm", label="Submit to Farm"),
+            NumberDef("priority", label="Farm job Priority", default=50),
+            BoolDef("refresh", label="Refresh viewport during export"),
+            BoolDef("includeParentHierarchy", label="Include Parent Hierarchy"),
+            BoolDef("writeNormals", label="Write Normals"),
```

Comment:
Why are these above the "Animation Options" label?

Reply:
They are above the Alembic options, since they are "generic" animation settings (`writeNormals` being an weird case, since it's used in the usd extractor ):
```
Generic Animation Settings
-----------------
Alembic Specific Settings
```